src/_guides/language/language-tour.md

```diff
@@ -108,7 +108,8 @@ mind:
     Recent changes to the Dart language and tools
     make writing type safe code possible,
     with many expected benefits;
-    see [Strong Mode Dart](/guides/language/sound-dart) for details.
+    see [Strong Mode Dart](/guides/language/sound-dart)
```

Comment:
We should have a separate bullet item that introduces strong mode. (It's referred to a few places in this page, but it's never defined.)